### ISSUE TITLE
Please keep Raspbian (arm32) support

### ISSUE THREAD

**Issue by Letme** [linuxserver/docker-unifi-controller#110]
I have read that due to lack of interest the arm32 bit is going to be deprecated, but officially all the Raspbian images are 32 bit. So I hope that alone makes enough sense to keep supporting. If there is some task needed, I am also willing to contribute.

------------------------------

## Alternatives Considered

None, I would not like to be  building the container myself.

**Comment by Roxedus** [comment]
It has nothing to with interest, and everything to do with upstream decisions. 

Right now armhf is held back to a version of Ubuntu which is End-of-life, due to there being now official mongodb package past that. 
As for OSes, both the Raspberry os 64bit beta and Ubuntu has worked greatly for us.

**Comment by danstreeter** [comment]
I recently upgraded from 32 bit rasbian to 64 on a 3b+ without any problem.
Now get no problems with the unifi controller container and mongo.

I scriped my base raspi configuration into an ansible playbook so I never had to do it manually again, feel free to take a look https://github.com/danstreeter/ansible-raspberrypi

Might speed up getting onto a 64bit instance - you'll just need to burn a 64 bit image to SD, enable ssh on the pi, run the playbook and then perform your install of the unifi-controller in docker on the pi.
